### DISCUSSION TITLE
Win 10 22H2 not supported?

### DISCUSSION THREAD

**Discussion by D4n2021** [File-New-Project/EarTrumpet#1372] [Q&A] [answered]
22H2 is missing here. Is this just a little accident or why is it not claimed to be supported? https://github.com/File-New-Project/EarTrumpet#supported-operating-systems

**Comment by riverar** [comment, marked as answer]
@D4n2021 Just an oversight on my part, good catch. Will fix.

Edit: Done. https://github.com/File-New-Project/EarTrumpet/commit/25364401ef9ad31fc88f978fa5d90934ff716b65

**Reply by D4n2021** [reply]
that was quick, awesome! Thanks!
